README.md

```diff
@@ -88,3 +99,13 @@ Irene->Bob: countersigned updated state
 Alice->Irene: countersigned updated state
 end
 -->
+
+### Future work
+
+This repo originated as an entry for the ETHGlobal 2023 hackathon. Therefore some shortcuts were taken to enable an end-to-end demo given the time constraints. The following list identifies future areas for improvement if this foundation was used to build a production-ready application.
+
+1. Messaging: currently in-browser `BroadcastChannels` are used to communicate between the different users. A proper messaging service should allow users to exchange messages when they are on different machines with different IP addresses. Possible technologies include: WebSockets, WebRTC, HTTP requests, etc.
+2. Key management: currently the private signing keys used by each user to sign blockchain txs and state channel messages are stored in a static [.env](./.env) file. Since these keys control the movement of funds, they should be stored more securely, i.e. in Metamask or another crypto wallet.
```

Comment:
Could mention those keys get statically inserted into the js running on the page.